dockerfiles/base/Dockerfile.ubi

```diff
@@ -91,8 +91,9 @@ EOF
 
 COPY --from=broker-builder /opt/app-root/src/.npm-global /opt/app-root/src/.npm-global
 COPY --from=node-base /tmp/node/bin/node /usr/bin/node
-COPY --chown=snyk:snyk config.default.json /home/snyk/config.default.json
-COPY --chown=snyk:snyk defaultFilters /home/snyk/defaultFilters
+COPY --chown=snyk:root config.default.json /home/snyk/config.default.json
+COPY --chown=snyk:root defaultFilters /home/snyk/defaultFilters
+RUN chown -R snyk:root /home/snyk
```

Comment:
Should we add this line here as well (as multiline to reduce amount of Docker layers :))?

https://github.com/snyk/broker/blob/master/dockerfiles/Dockerfile.ubi#L11

Reply:
I think the SCM-specific config is best left in the `dockerfiles/Dockerfile.ubi` file personally - means we don't have to rebuild `/dockerfiles/base/Dockerfile.ubi` container many times (unless I'm misunderstanding the comment!).

Reply:
You're right, we don't rebuild base container many times.

What I mean was: we'll need the same `chown snyk:root` command in `dockerfiles/Dockerfile.ubi` file because we generate accept.json there.